content/components/selectpanel.mdx

```diff
@@ -64,58 +160,269 @@ Select panel should be anchored to its trigger.
+      src="https://github.com/primer/design/assets/980622/4e11eb8a-c515-49c8-9b48-75330b72a963"
+    />
+    <Caption>Display a clear icon within the input to clear it's value.</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a click action outside of the dialog."
+      src="https://github.com/primer/design/assets/980622/079feed7-4ddc-476a-b639-ea17de141b9d"
+    />
+    <Caption>Don't forget to display a clear icon when the input has a value.</Caption>
+  </Dont>
+</DoDontContainer>
+
+For multi-selections with search functionality, a convenient **deselect all** [icon button](/components/icon-button) becomes available as soon as at least one item is selected.
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a clicking action on the apply button."
+      src="https://github.com/primer/design/assets/980622/381b2224-3ad0-4320-94cf-8284eb3f5157"
+    />
+    <Caption>
+      When one or more items are selected, display a deselect all button in the header to deselect the entire selection.
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a click action outside of the dialog."
+      src="https://github.com/primer/design/assets/980622/af95eaee-976b-48ab-8fb2-b31389318fca"
+    />
+    <Caption>When one or more items are selected don't forget to display a deselect all button in the header.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Action menu trigger
+
+When the select panel is activated through an [action menu](/components/action-menu), it should be displayed as a modal dialog. It should never be shown as a submenu.
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a clicking action on the apply button."
+      src="https://github.com/primer/design/assets/980622/22386756-2612-40dd-aceb-1f765095cf17"
+    />
+    <Caption>When the select panel is triggered by an action menu, it should be presented as a modal dialog.</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a click action outside of the dialog."
+      src="https://github.com/primer/design/assets/980622/a71c59d6-ca35-49ac-b082-e72aa4d1a8cd"
+    />
+    <Caption>Avoid displaying the select panel as a non-modal submenu within an action menu.</Caption>
   </Dont>
 </DoDontContainer>
 
 ## Behavior
 
+### Appearance
+
+<img width="1338" alt="" src="https://github.com/primer/design/assets/980622/28aeeca1-01d7-444f-964f-6a2744c1ced5" />
+
+The select panel can appear as a nonmodal dialog or modal dialog.
+
+- **Nonmodal dialog**: When triggered by a [button](/components/button) or [icon button](/components/icon-button) but never by an [action menu](/components/action-menu).
+- **Modal dialog**: When triggered by a [button](/components/button), [icon button](/components/icon-button) or [action menu](/components/action-menu).
```

Comment:
Small nit, the terms non-modal dialog means that the dialog stays on the screen even when interacting outside of it, and modal means that you can't interact with items outside of it before closing it. Both or neither can have different positional anchoring on the screen.

Reply:
No nit at all! I meant hereL

```
The select panel can appear as an anchored dialog or floating dialog.

- **Anchored dialog**: When triggered by a [button](/components/button) or [icon button](/components/icon-button) but never by an [action menu](/components/action-menu).
- **Floating dialog**: When triggered by a [button](/components/button), [icon button](/components/icon-button) or [action menu](/components/action-menu).
```